src/client_side_request.cc

```diff
@@ -943,7 +943,7 @@ clientInterpretRequestHeaders(ClientHttpRequest * http)
         }
     }
 
-    /* Only HEAD and GET requests permit a Range or Request-Range header.
+    /* Only HEAD, GET, and QUERY requests permit a Range or Request-Range header.
```

Comment:
Please avoid detailed retelling of the code (that requires updates and creates stale comments):

```suggestion
    /* Only certain requests permit a Range or Request-Range header.
```

or

```suggestion
    /* Only some requests permit a Range or Request-Range header.
```

or something like that.

Reply:
Done in 33b2abf6dd